src/snowflake/snowpark/mock/_plan.py

```diff
@@ -362,6 +363,11 @@ def handle_function_expression(
                 exp, input_data, analyzer, expr_to_alias, current_row
             )
 
+        if exp.api_call_source == "functions.call_udf":
+            raise SnowparkLocalTestingException(
+                f"Unknown function {func_name}. UDF by that name does not exist."
+            )
```

Comment:
can you help me understand the relationship between the if condition and udf not existing?

Reply:
I see, so the if case is for `call_udf()` and the else case is for `builtin()`?

Reply:
yep